src/main.js

```diff
@@ -15,6 +15,19 @@
 var Util = require('./util.js');
 var WebVRPolyfill = require('./webvr-polyfill.js').WebVRPolyfill;
 
+window.WebVRPolyfillMode = {
```

Comment:
Do we need this distinction? Why are the non-zero modes required?

Also, I don't think this needs to be global. Looks like you expect configuration to be set via WebVRConfig, so this variable can be internal to the polyfill.